### ISSUE TITLE
Adding an extra class to a pretrained SSD model

### ISSUE THREAD

**Issue by miladf2** [weiliu89/caffe#539]
I am trying to add a new class to VGG_ILSVRC_16_layers_fc_reduced which was trained on pascal voc data to detect 20 classes. I get the following error

``` F0417 21:06:13.156370 22920 detection_output_layer.cpp:166] Check failed: num_priors_ * num_classes_ == bottom[1]->channels() (540408 vs. 515844) Number of priors must match number of confidence predictions.```

I dont exactly know what bottom[1]->channels() is referring to, but I guess it has to be the last layer of the network.

**Comment by weiliu89** [comment]
I don't understand how you add the new class. You should finetune the model again for the newly added class. The error means that you specify 22 classes in the DetectionOutputLayer, but actually the model only produces 21 classes predictions.

**Comment by miladf2** [comment]
I dont exactly know how to fine-tune the model in Caffe because there is no documentation online about adding a class. Is there a resource on fine-tuning? Can you point me to the files that I need to change?

**Comment by miladf2** [comment]
It actually worked. The problem wasnt adding extra class... problem was that I picked a large learning rate causing the algorithm to diverge. I reduced the learning rate and it worked.
